apps/storybook-react-native/src/index.test.ts

```diff
@@ -0,0 +1,9 @@
+import greeter from '.';
```

Comment:
non-blocking: Can be removed as this wont be a published package

Reply:
can you clarify?

Reply:
Will this `index.test.ts` file be required? It's removed from the design-system-react-native package so I don't think we will need it but it can be removed later? https://github.com/MetaMask/metamask-design-system/pull/50/files#diff-3f0cc379008ecb010632c3a6480783ce754def4f4329fc0a178cd38c48eebc90